pkg/controller/controller.go

```diff
@@ -98,7 +100,7 @@ func (c *Controller) Run(threadiness int, stopCh <-chan struct{}) error {
 	if !cache.WaitForCacheSync(stopCh, c.podInformer.HasSynced) {
 		return fmt.Errorf("controller.Run: Timeout waiting for pod cache sync")
 	}
-	glog.Infoln("Controller cache has synced")
+	glog.V(Vdebug).Infoln("Controller cache has synced")
```

Comment:
Could probably be `Vadmin` level

Reply:
OK, but pushing back a little :)  Why is this something an admin cares about vs. someone who needs to figure out why the controller isn't working (debugging)?

Reply:
I can't really provide a specific reason, it just feels more Admin-ish :P I'm fine with leaving as it though.